README.md

```diff
@@ -29,6 +29,8 @@ A possible use case for this is to be able to use different Z-Wave networks, on
 
 ## Installation
 
+This addon should be installed on the master instance of Home Assistant
```

Comment:
We should use *integration* instead of *addon* as the latter is the terminology used by the supervisor. Also change *master* to *main*.

Reply:
Interestingly, the documentation here describes this as a *component*.

Reply:
I believe that is technically wrong, but the terminology is a bit confusing to say the least. I'm still surprised we call it custom components, rather than custom integrations TBH.